experimental/IntersectionTheory/docs/src/intro.md

```diff
@@ -6,44 +6,135 @@ DocTestSetup = Oscar.doctestsetup()
-	Chow rings. These rings are graded by the codimension of cycles.
+    TODO: Why numerical intersection rings?
 
-As in `Schubert`, our approach is abstract in the sense that we do not work with explicit
-varieties given by equations. Instead, we represent a variety by specifying its
-dimension together with its Chow ring and, possibly, further data. We refer to such
+With respect to computations, again following `Schubert`, our approach is abstract in the
+sense that we do not work with explicit varieties given by equations. Instead, we specify the
+dimension of the variety together with its Chow ring and, possibly, further data. We refer to such
 a collection of data as an *abstract variety*, and to results obtained from manipulating
 the data as results which apply to all (smooth projective complex) varieties sharing the data. 
 
-Of particular interest is the tangent bundle of a variety (recall that the Todd class of the
+Of particular interest is the tangent bundle of a variety (the Todd class of the
 tangent bundle enters the Hirzebruch-Riemann-Roch formula). As with any other vector bundle,
 the tangent bundle is represented as a collection of data referred to as an *abstract vector bundle*.
-The main data here is the Chern character polynomial of the vector bundle.
+The main data here is the Chern character of the vector bundle.
+
+In the same spirit, we introduce  *abstract variety maps*. Their key data is the pullback 
+morphism between the respective Chow rings.
+
+Some comments on this set-up are in order:
+
+Let $X$ be a variety. Write $K^{\circ}(X)$  for the *Grothendieck group of vector bundles on $X$*. That is,
+$K^{\circ} (X)$ is the free abelian group generated by the isomorphy classes of vector bundles on $X$,
+modulo the relations of type $[E]=[E^{\prime}]-[E^{\prime \prime}]$, where $E$, $E^\prime$, and
+$ E^{\prime \prime}$ fit into an exact sequence
+$0\rightarrow E^\prime \rightarrow E \rightarrow E^{\prime \prime} \rightarrow 0$.
+The tensor product turns $K^{\circ}(X)$ into a ring,  the *Grothendieck ring of vector bundles on $X$*.
+By Riemann-Roch, the Chern character defines a ring isomorphism $\text{ch}: K^{\circ}(X)_{\mathbb Q}
+\overset{\simeq}\longrightarrow A^*(X) _{\mathbb Q}$, where $A^*(X)$ is obtained by considering cycles
+modulo rational equivalence. Since rational equivalence is (much) finer than
+numerical equivalence, we obtain a ring epimorphism
+
+$\text{ch}: K^{\circ}(X)_{\mathbb Q}
+\overset{\simeq}\longrightarrow A^*(X) _{\mathbb Q}\twoheadrightarrow N^*(X)_{\mathbb Q}.$
+
+!!! note
+    With notation as above, write $K_{\circ} (X)$  for the *Grothendieck group of coherent sheaves on $X$*.
+    Then, since $X$ is supposed to be smooth, the natural map $K^{\circ} (X)\rightarrow K_{\circ} (X)$ is an isomorphism
+	(every coherent sheaf on $X$ has a finite resolution by locally free sheaves). We may, thus, speak of coherent sheaves
```

Comment:
There are some tabs at multiple places in this file. Please convert them to spaces